ui/app/index.html

```diff
@@ -180,34 +180,45 @@
-          </li>
-          <li ui-sref-active="active">
-            <a href ui-sref="artifacts.list">Artifacts</a>
+          <li class="dropdown" ui-sref-active="active">
+            <a href id="configs-dropdown-button" class="dropdown-toggle" data-toggle="dropdown" role="button"
+               aria-expanded="false">Configs<span class="caret"></span></a>
+            <ul class="dropdown-menu" role="menu">
+              <li ui-sref-active="active"><a href ui-sref="build-configs.list">Build Configs</a></li>
+              <li ui-sref-active="active"><a href ui-sref="group-configs.list">Group Configs</a></li>
+            </ul>
           </li>
           <li class="dropdown" ui-sref-active="active">
-            <a href id="builds-dropdown-button" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Builds<span class="caret"></span></a>
+            <a href id="builds-dropdown-button" class="dropdown-toggle" data-toggle="dropdown" role="button"
+               aria-expanded="false">Builds<span class="caret"></span></a>
             <ul class="dropdown-menu" role="menu">
               <li ui-sref-active="active"><a href ui-sref="builds.list">Builds</a></li>
               <li ui-sref-active="active"><a href ui-sref="group-builds.list">Group Builds</a></li>
             </ul>
           </li>
           <li ui-sref-active="active">
-            <a href ui-sref="build-configs.list">Build Configs</a>
+            <a href ui-sref="artifacts.list">Artifacts</a>
           </li>
           <li ui-sref-active="active">
             <a href ui-sref="scm-repositories.list">SCMs</a>
           </li>
           <li class="dropdown" ui-sref-active="active">
-            <a href class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Reports<span class="caret"></span></a>
+            <a href class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">Reports<span
+              class="caret"></span></a>
             <ul class="dropdown-menu" role="menu">
-              <li ui-sref-active="active"><a href ui-sref="report.product-shipped">Whitelisted artifacts of products</a></li>
-              <li ui-sref-active="active"><a href ui-sref="report.products-for-artifact">Products for whitelisted artifacts</a></li>
-              <li ui-sref-active="active"><a href ui-sref="report.blacklisted-artifacts-in-project">Blacklisted artifacts in a project</a></li>
-              <li ui-sref-active="active"><a href ui-sref="report.different-artifacts-in-products">Different artifacts between products</a></li>
-              <li ui-sref-active="active"><a href ui-sref="report.built-artifacts-in-project">Built artifacts in a project</a></li>
-              <li ui-sref-active="active"><a href ui-sref="report.project-product-diff">Diff a project against a product</a></li>
+              <li ui-sref-active="active"><a href ui-sref="report.product-shipped">Whitelisted artifacts of products</a>
```

Comment:
Just very minor comment about formatting, I would say readability of something like:
```
<li>
    <a>lorem ipsum</a>
</li>
```
would be better than the current one:
```
<li><a>lorem
    ipsum</a></li>
```